.env.komenci

```diff
@@ -0,0 +1,207 @@
+AZURE_SOUTHBR_KOMENCI_DB_PORT=5432
+AZURE_SOUTHBR_KOMENCI_DB_USERNAME=postgres@mainnet-komenci-brazil
+AZURE_SOUTHBR_KOMENCI_DB_PASSWORD_VAULT_NAME=mainnet-komenci-brazil
+
+# Recaptcha token
+AZURE_EASTUS_KOMENCI_RECAPTCHA_SECRET_VAULT_NAME=staging-komenci-eus
+AZURE_WESTEU_KOMENCI_RECAPTCHA_SECRET_VAULT_NAME=staging-komenci-weu
+AZURE_SOUTHBR_KOMENCI_RECAPTCHA_SECRET_VAULT_NAME=mainnet-komenci-brazil
+
+# Rule config > Captcha
+KOMENCI_RULE_CONFIG_CAPTCHA_BYPASS_TOKEN=special-captcha-bypass-token
+AZURE_EASTUS_KOMENCI_RULE_CONFIG_CAPTCHA_BYPASS_ENABLED=true
+AZURE_WESTEU_KOMENCI_RULE_CONFIG_CAPTCHA_BYPASS_ENABLED=true
+AZURE_SOUTHBR_KOMENCI_RULE_CONFIG_CAPTCHA_BYPASS_ENABLED=false
+
+# Format should be a comma-separated sequence of:
+# <address>:<key vault name>:<resource group (optional)>
+#AZURE_EASTUS_KOMENCI_ADDRESS_AZURE_KEY_VAULTS=0x21860ca3a0a6f7e450b8f24bd00eac7ba766b85e:testnet-oracle-eastus,0xfd3738e5e0a020614a9e5253078dda491e77031c:testnet-oracle-eastus
+AZURE_EASTUS_KOMENCI_ADDRESS_AZURE_KEY_VAULTS=0x00454cac6dae53f8800f71395b9a174f07a784b1:staging-komenci-eus,0xc6f0f9bfb1aed83620ece3eac0add98a65a8574e:staging-komenci-eus
+AZURE_WESTEU_KOMENCI_ADDRESS_AZURE_KEY_VAULTS=0x0f812be74511b90ea6b2f80e77bea047e69a0b2a:staging-komenci-weu,0xb354d3d2908ba6a2b791683b0f454a38f69cb282:staging-komenci-weu
+AZURE_SOUTHBR_KOMENCI_ADDRESS_AZURE_KEY_VAULTS=0x21888ae301658cdff7ce8c33cdf83a330a5e6273:mainnet-relayer0,0x1438128a2dcc645f0b9706350c1f5dad04845fe6:mainnet-relayer1,0x1e36bf42272a0693eba69332a6f623ce37694a27:mainnet-relayer2,0xd5afaaa7256c9eb86376c4214635dd56dffbd3a8:mainnet-relayer3,0xb09eba8bc1c8bedadd634a8219c0b09042170903:mainnet-relayer4
+AZURE_VAULT_NAME=0x00454cac6dae53f8800f71395b9a174f07a784b1:staging-komenci-eus,0xc6f0f9bfb1aed83620ece3eac0add98a65a8574e:staging-komenci-eus
+
+# Network
+AZURE_SOUTHBR_KOMENCI_NETWORK=rc1
+AZURE_EASTUS_KOMENCI_NETWORK=alfajores
+AZURE_WESTEU_KOMENCI_NETWORK=alfajores
+
+AZURE_EASTUS_KOMENCI_ADDRESSES_FROM_MNEMONIC_COUNT
+AZURE_EASTUS_FULL_NODES_COUNT=2
+AZURE_EASTUS_FULL_NODES_DISK_SIZE=10
+AZURE_WESTEU_KOMENCI_ADDRESSES_FROM_MNEMONIC_COUNT
+AZURE_WESTEU_FULL_NODES_COUNT=2
+AZURE_WESTEU_FULL_NODES_DISK_SIZE=10
+AZURE_SOUTHBR_FULL_NODES_COUNT=2
+AZURE_SOUTHBR_FULL_NODES_DISK_SIZE=10
+
+AWS_TEST_AWS_KUBERNETES_CLUSTER_REGION=us-west-2
+AWS_TEST_AWS_KUBERNETES_RESOURCE_GROUP=adorable-monster-1597251246
+AWS_TEST_KUBERNETES_CLUSTER_NAME=adorable-monster-1597251246
+AWS_TEST_FULL_NODES_COUNT=2
+AWS_TEST_FULL_NODES_DISK_SIZE=10
+
+GCP_TEST_GCP_PROJECT_NAME=celo-testnet
+GCP_TEST_GCP_ZONE=us-west4-a
+GCP_TEST_KUBERNETES_CLUSTER_NAME=federated-dev-us-west4-a
+GCP_TEST_FULL_NODES_COUNT=1
+GCP_TEST_FULL_NODES_DISK_SIZE=10
+
+GCP_TEST_ASIA_GCP_PROJECT_NAME=celo-testnet
+GCP_TEST_ASIA_GCP_ZONE=asia-northeast2-a
+GCP_TEST_ASIA_KUBERNETES_CLUSTER_NAME=federated-dev-asia-northeast2-a
+GCP_TEST_ASIA_FULL_NODES_COUNT=1
+GCP_TEST_ASIA_FULL_NODES_DISK_SIZE=10
+
+# ---- General ----
```

Comment:
Everything from here down seams dangerous in the sense that I'm not sure why it lives here and if it clashes with some other env vars in some other files and if they override each other.